playback-sdk-android/src/main/java/com/streamamg/api/player/PlaybackAPIService.kt

```diff
@@ -51,6 +52,10 @@ internal class PlaybackAPIService(private val apiKey: String) : PlaybackAPI {
                     setRequestProperty("Authorization", "Bearer $authorizationToken")
                 }
 
+                if (!userAgent.isNullOrBlank()) {
```

Comment:
@StefanoStream Would there be a case where user agent can be null, if so what should be the flow, since without user agent set the API fails?

Reply:
@artem-y-pamediagroup are you able to answer to @zachtom question?

Reply:
As discussed with @StefanoStream, to unblock todays release we can pick this up later when @artem-y-pamediagroup is back.